app/content/models/user.py

```diff
@@ -111,9 +111,13 @@ def has_retrieve_permission(cls, request):
 
     @classmethod
     def has_list_permission(cls, request):
-        return check_has_access(cls.has_access, request) or len(
-            request.user.membership.filter(membership_type=MembershipType.LEADER)
-        )
+        try:
+
+            return check_has_access(cls.has_access, request) or len(
+                request.user.membership.filter(membership_type=MembershipType.LEADER)
```

Comment:
Ser at related name er blitt fjernet fra Membership-modellen så det kan være grunnen til at `.membership` er None. Ville lagt til related name og brukt den her

Reply:
den er på membership men ikke membership history, endret til memberships nå da

Reply:
erroren kommer at at request user er none hvis man ikke er logget inn og sjekker groups for da er memberships none siden user er none

Reply:
Er det en måte å blokkere en uautorisert request tidligere?